week-4-tests-and-persistence/w04e01-simple-testing/src/app/divider.js

```diff
@@ -2,5 +2,8 @@
  * Tę funkcjonalność chcemy przetestować:
  * */
 export function divider(dividend, divisor) {
-  return dividend / divisor
+  if (typeof divisor !== "number") throw new Error("Divisor must be a number");
+  if (typeof dividend !== "number") throw new Error("Dividend must be a number");
```

Comment:
teraz jak masz testy, to uruchamiasz je w `watch` mode.

I refaktoryzujesz ten kod.

Zobacz na to powtórzenie:
```javascript
 if (typeof divisor !== "number") throw new Error("Divisor must be a number");
 if (typeof dividend !== "number") throw new Error("Dividend must be a number");
```

Co możemy z tym zrobić 🤔??

Reply:
Wyjęte do osobnej funkcji 👌